Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:lts-alpine as BUILD
+
+ARG ENVIRONMENT=development
+WORKDIR /app
+
+COPY . .
+RUN yarn install --frozen-lockfile
+RUN NODE_ENV=$ENVIRONMENT yarn run env-gen
+RUN yarn build
+
+# ------------------------------------------------------------
+FROM node:lts-alpine as APP
+
+WORKDIR /app
+
+COPY --from=BUILD /app/build ./build
+COPY --from=BUILD /app/build/package.json .
+COPY --from=BUILD /app/yarn.lock .
+COPY --from=BUILD /app/.env .
+COPY --from=BUILD /app/ecosystem.config.js .
+
+RUN yarn install --production
```

Comment:
I think this should be left out here. This stage is for the build output, while the one above is for doing the build. Id add a copy step for the node_modules from the build stage to this stage and leave this layer out